### DISCUSSION TITLE
表格的边距调整

### DISCUSSION THREAD

**Discussion by ujust4fun** [sjtug/SJTUThesis#974] [提问] [answered]
表格Threeparttable环境左右太长了，怎么样左移一下表格使页面的左边边距可以适当减少呢
![image](https://github.com/sjtug/SJTUThesis/assets/25298124/1c4901ed-aacf-4e57-abda-24d234070cd7)

**Comment by LogCreative** [comment, marked as answer]
```latex
\begin{table}[htp]
    \caption{表格标题}
    \centering
    \hspace*{-5cm} % 基于中间位置的偏移量
    \makebox[0cm]{ %
        \begin{threeparttable}
            \begin{tabular}{cccc}
            % ...
            \end{tabular}
        \end{threeparttable}
    } %
\end{table}
```

可以突破左边距位置，但是一般不推荐这么做。

<img width="404" alt="截屏2023-07-01 20 01 41" src="https://github.com/sjtug/SJTUThesis/assets/61653082/a7ab1cd4-55c8-4ce3-b813-92ff6f44c944">

**Reply by ujust4fun** [reply]
谢谢。用其它办法间接实现了
